### PR TITLE
[DOCS-6383] Rewrite review intro page

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
Rewrote: search-enterprise/latest/index.md 
This is the only page that requires review, the other pages are being reviewed using other PRs.

### CODE REVIEW

**Review by mjstanford — APPROVED** [review]
Looks good.

File: search-enterprise/latest/install/index.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+--spring.datasource.password=alfresco \
+--alfresco.reindex.prefixes-file=file:reindex.prefixes-file.json
+o.s.batch.core.step.AbstractStep         : Step: [reindexByIdsStep] executed in 4s952ms
+o.a.r.w.ElasticsearchRepoEventItemWriter : Total indexed documents:: 845
+o.a.r.listeners.JobLifecycleListener     : Current Status: COMPLETED
+```
+
+Once the program has being executed, existing Alfresco Repository nodes will be available in Elasticsearch.
+
+> **Note:** Additional use cases for this application will be covered in the [Indexing](Indexing) section.
```
Comment (gravitonian) [inline search-enterprise/latest/install/index.md:141]: Incorrect link: [Indexing](Indexing)

File: search-enterprise/latest/config/index.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+ java -jar target/alfresco-elasticsearch-reindexing-3.0.0-SNAPSHOT-app.jar \
+  --alfresco.reindex.jobName=reindexByDate \
+  --alfresco.reindex.pagesize=100 \
+  --alfresco.reindex.batchSize=100  \
+  --alfresco.reindex.concurrentProcessors=6 \
+  --alfresco.reindex.fromTime=202001010000 \
+  --alfresco.reindex.toTime=202104180000
+```
+
+> **Note:** Additional use cases for this application will be covered in the [Indexing](Indexing) section.
```
Comment (gravitonian) [inline search-enterprise/latest/config/index.md:146]: Incorrect link: [Indexing](Indexing)

File: search-enterprise/latest/admin/index.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+## Searching performance
+
+Alfresco Elasticsearch Connector is using an out-of-the-box Elasticsearch server. In order to improve query performance, [official recommendations from Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/tune-for-search-speed.html) are valid.
+
+## Reindexing
+
+Reindexing component has been tested on reading replica with Postgres database.
+Tests have been performed on the local environment and AWS.
+
+To run database read replicas in AWS follow the guidelines given in [AWS read replicas](https://aws.amazon.com/rds/features/read-replicas/l)
```
Comment (gravitonian) [inline search-enterprise/latest/admin/index.md:241]: Incorrect URL

File: search-enterprise/latest/admin/index.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+
+```bash
+spring.datasource.url=jdbc:postgresql://<READ_REPLICA_ADDRESS>:<READ_REPLICA_PORT>/alfresco
+```
+
+## Reindex using remote partitioning
+
+When it is required to index a huge Alfresco Repository instance, using a single reindex process can require much time. We can scale the reindex node vertically in order to improve performance, but this cannot be enough. Using [remote partitioning](https://docs.spring.io/spring-batch/docs/current/reference/html/scalability.html#partitioning), a Spring Batch feature, we can scale horizontally the Reindexing service.
+
+
```
Comment (gravitonian) [inline search-enterprise/latest/admin/index.md:253]: Following picture need to be formatted as code

File: search-enterprise/latest/using/index.md [hunk 5]
```diff
@@ -0,0 +1,750 @@
...
+
+Any character can be escaped using the backslash "" in terms, IDs (field identifiers), and phrases. Java unicode escape sequences are supported. Whitespace can be escaped in terms and IDs.
+
+For example:
+
+```sql
+cm:my content:my name
+```
+
+[Search using Type](https://docs.alfresco.com/content-services/latest/develop/rest-api-guide/searching)#searching-by-content-type-and-controlling-paging-and-sorting
```
Comment (gravitonian) [inline search-enterprise/latest/using/index.md:160]: Link not taking you to the correct place I think

File: search-enterprise/latest/using/index.md [hunk 6]
```diff
@@ -0,0 +1,750 @@
...
+
+```sql
+="taxi driver"
+```
+
+> **Note:** Exact Term Search is disabled by default, to enable it please refer to Indexing documentation and the configuration file: exactTermSearch.properties
+
+## Searches that involve stopwords
+
+[Stopwords](_https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-stop-tokenfilter.html#analysis-stop-tokenfilter-stop-words-by-lang_){:target="_blank"} are removed from the query.
```
Comment (gravitonian) [inline search-enterprise/latest/using/index.md:366]: Incorrect link.

File: search-enterprise/latest/using/index.md [hunk 7]
```diff
@@ -0,0 +1,750 @@
...
+|`[#1 TO #2>`| |The range #1 to #2 including #1 but not #2.`#1 <= x < #2`|`[0 TO 5>`|
+|`<#1 TO #2>`| |The range #1 to #2 exclusive.`#1 < x < #2`|`<0 TO 5>`|
+
+```sql
+TEXT:apple..banana
+my:int:[0 TO 10]
+my:float:2.5..3.5
+my:float:0..MAX
+mt:text:[l TO "uFFFF"]
+
```
Comment (gravitonian) [inline search-enterprise/latest/using/index.md:556]: Missing end of formatting block

**Review by gravitonian — CHANGES_REQUESTED** [review]
A few link changes etc needed

File: search-enterprise/latest/install/index.md [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+
+## Alfresco Re-Indexing app
+
+Since Alfresco Elasticsearch Connector *Live Indexing* component is listening to messages from ActiveMQ, some initial information from Alfresco Repository needs to be indexed using the *Re-Indexing* component. This *Re-Indexing* component can be also used to index a pre-populated Alfresco Repository.
+
+Before running this application, it's required to get a JSON mapping of namespace to prefix for deployed content models. You may use the following project to build this JSON file:
+
+[Alfresco Model Namespace-Prefix Mapping](https://github.com/AlfrescoLabs/model-ns-prefix-mapping){:target="_blank"}
+
+Copy the [JAR deployment](https://github.com/AlfrescoLabs/model-ns-prefix-mapping/releases/download/1.0.0/model-ns-prefix-mapping-1.0.0.jar){:target="_blank"} file for this module from to your local Alfresco Repository deployment. You can find instructions on how to deploy a Simple JAR Alfresco Repository module in [Simple Module (JAR)](https://docs.alfresco.com/content-services/2.0/develop/extension-packaging/#simplemodule){:target="_blank"}.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply search-enterprise/latest/install/index.md:87]: this page is a part of another PR

File: search-enterprise/latest/config/index.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/config/index.md:146]: this page is a part of another PR

File: search-enterprise/latest/admin/index.md (hunk 4 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/admin/index.md:253]: this page is a part of another PR

File: search-enterprise/latest/using/index.md (hunk 6 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/using/index.md:366]: this page is a part of another PR

File: search-enterprise/latest/using/index.md (hunk 7 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/using/index.md:556]: this page is a part of another PR

File: search-enterprise/latest/admin/index.md (hunk 3 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/admin/index.md:241]: this page is a part of another PR

File: search-enterprise/latest/install/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/install/index.md:141]: this page is a part of another PR

**Comment by oliveralfresco2018** [comment]
@gravitonian  I have not corrected any of the pages as they are part of other PRs. 
I have closed them as resolved. 
Let me know if there are any suggestions for the intro page. If not can you approve this? Cheers

**Pull request merged**